assets/css/index.css

```diff
@@ -9,6 +9,30 @@ body {
   color: #333;
 }
 
+[data-theme="dark"] {
+  background-color: #131313;
+  color: #ffffff;
+}
+
+[data-theme="dark"] footer, .navbar.scrolling {
+  background-color: #000000;
```

Comment:
Change footer background to `#434343`

Reply:
And the navbar stays #000000 or should I change it too?

Reply:
Change it too